src/application/appwindow.cpp

```diff
@@ -253,31 +251,6 @@ void AppWindow::loadCustomCSS()
                                                GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
 }
 
-void AppWindow::disableEditingActions()
-{
-    m_openAction->set_enabled(false);
-    m_saveAction->set_enabled(false);
```

Comment:
The "Save" action should still be disabled while saving and re-enabled after saving, since we can't show two in-app notifications for this.

It should be enough to replace the `disableEditingActions` and `enableEditingActions` calls with disabling and enabling the "Save" action.